examples/dataproc-persistent-history-server/terraform/service-account.tf

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_service_account" "history-server-account" {
+  project    = "${var.project}"
+  account_id = "history-server-account"
+}
+
+resource "google_project_iam_member" "history-server-dataproc-worker-role" {
+  project = "${var.project}"
+  role    = "roles/dataproc.worker"
+
+  member = "serviceAccount:history-server-account@${var.project}.iam.gserviceaccount.com"
```

Comment:
```suggestion
  member = "${google_service_account.history-server-account.email}"
```

Reply:
Done. FYI needed to have `serviceAccount:` prefix.